zfit/core/integration.py

```diff
@@ -378,8 +374,8 @@ def mc_vf_integrate(
     n_events: int = int(6e6),
     events_limit: int = int(1e6),
     list_devices: list = None,
-    compilable: bool = True,
-    verbose: bool = False,
+    compilable: bool = False,
+    verbose: int | str = 3,
```

Comment:
```suggestion
    verbose: int | str = 0,
```
Let the default be quiet